<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_exemplaire</class>
 <widget class="QDialog" name="Dialog_exemplaire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>511</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(44, 62, 89);
border-right-color: rgb(44, 62, 89);
border-color: rgb(44, 62, 89);
selection-background-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 255, 255);
border-bottom-color: rgb(44, 62, 89);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_Ajouter">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_libelleexemplaire_2">
     <property name="geometry">
      <rect>
       <x>192</x>
       <y>142</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id_exemplaire">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="ide_exemplaire">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>124</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Identifiant exemplaire</string>
     </property>
    </widget>
    <widget class="QLabel" name="libelle_exemplaire_2">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>142</y>
       <width>107</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Libelle exemplaire </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ajouter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>260</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Supprimer">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>58</y>
       <width>450</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Vous devez Tapez l'identifiant d'exemplaire que vous souhaite le supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_exemplairsupp">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>100</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>valider</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Modifer">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>60</y>
       <width>410</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Vous devez Tapez l'identifiant d'exemplaire que vous souhaite modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ide_modif">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QLabel" name="libelle_exemplaire">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>172</y>
       <width>107</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Libelle exemplaire </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_libelleexemplaire">
     <property name="geometry">
      <rect>
       <x>202</x>
       <y>172</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Consulter">
    <attribute name="title">
     <string>Consulter</string>
    </attribute>
    <widget class="QTableView" name="tableexemplaire">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>431</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
